src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java

```diff
@@ -388,6 +391,48 @@ public Long call() throws Exception {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @NonNull
+    public List<BitbucketServerTag> getTags() throws IOException, InterruptedException {
+        String url = String.format(API_TAGS_PATH, getUserCentricOwner(), repositoryName, 0, DEFAULT_PAGE_LIMIT);
```

Comment:
Need to encode the various URL segment parameters correctly, e.g. some will need to be `Util.rawEncode`, some will need to be `URLEncoder.encode` and you'll also need to check what happens for a tag that includes `/` in the name, does the `/` get encoded to `%2F` or does it stay as a `/` (and Bitbucket is not consistent in this regard, so you will need to test